plugin-gradle/CHANGES.md

```diff
@@ -7,6 +7,7 @@ We adhere to the [keepachangelog](https://keepachangelog.com/en/1.0.0/) format (
 ## [3.30.0] - 2020-05-11
 ### Added
 * `-PspotlessIdeHook` which makes the VS Code extension faster and more reliable.  See [`IDE_INTEGRATION.md`](IDE_INTEGRATION.md) for more details. ([#568](https://github.com/diffplug/spotless/pull/568))
+* Improve suggested gradle invocation for running `spotlessApply`. ([#578](https://github.com/diffplug/spotless/pull/578))
```

Comment:
Move this to the `[Unreleased]` section - it's too late to get included in `3.3.0` since it already shipped ;-)

Reply:
Done